pio/pio_pwmin.py

```diff
@@ -0,0 +1,98 @@
+from machine import Pin, PWM
+from rp2 import PIO, StateMachine, asm_pio
+from time import sleep, ticks_ms, ticks_diff
+
+pwm_out = PWM(Pin(16))
+
+pwm_out.freq(100)
+pwm_out.duty_u16((2**16-1)//2)
+
+@asm_pio()
+def pwmin():
+    pull(block)           # wait for activation
+    
+    set(x, 0)               # Set x = 0
+    mov(x, x | (0b01 << 3)) # invert x = Max-Value for 32 bits. (0b01 << 3) sets the invert bit.
```

Comment:
I think you could just do MOV x with a NULL source and the invert bit?

Reply:
Yes, that works.
Great Idea - thanks!